easybuild/tools/modules.py

```diff
@@ -1353,7 +1356,7 @@ def modules_tool(mod_paths=None, testing=False):
         modules_tool_class = avail_modules_tools().get(modules_tool)
         return modules_tool_class(mod_paths=mod_paths, testing=testing)
     else:
-        return None
+        return NoModulesTool(mod_paths=mod_paths, testing=testing)
```

Comment:
@MisterFruits Please change this to use positive logic, and avoid the duplicate stuff:

```python
if modules_tool is None:
    modules_tool_class = NoModulesTool
else:
    modules_tool_class = avail_modules_tools().get(modules_tool)
return modules_tool_class(mod_paths=mod_paths, testing=testing)
```

Another option is to simplify this a bit and use `NoModulesTool` as a fallback:

```python
modules_tool = get_modules_tool()
modules_tool_class = avail_modules_tools().get(modules_tool, NoModulesTool)
return modules_tool_class(mod_paths=mod_paths, testing=testing)

Reply:
I like the second solution =)
I named the argument in 690523d so it's more explicit.

Reply:
Okay, there is no named argument here... rolling back